doc/cephfs/cache-size-limits.rst

```diff
@@ -0,0 +1,15 @@
+Understanding MDS Cache Size Limits
+===================================
+
+This section describes ways to limit MDS cache size.
+
+You can limit the size of the Metadata Server (MDS) cache by:
+
+* *A memory limit*: A new behavior introduced in the {product} 3. Use the `mds_cache_memory_limit` parameters. Red Hat recommends to use memory limits instead of inode count limits.
```

Comment:
In upstream docs, we refer to Ceph by its release name (Luminous in this case).

Reply:
Also, use "We recommend" instead of "Red Hat".

Reply:
Ah, you are completely right, I've missed that.. The new version should be ok :)